src/tools/pub/index.md

```diff
@@ -1,14 +1,25 @@
 ---
-title: Pub Package and Asset Manager
-description: Use the pub tool to manage Dart's packages and assets.
+title: Pub Package Manager
```

Comment:
Same here with adding an aside about the historical function of `pub` and where to find modern alternatives (like, I assume, `build_runner`).